git/refs/reference.py

```diff
@@ -94,10 +94,10 @@ def name(self):
         return '/'.join(tokens[2:])
 
     @classmethod
-    def iter_items(cls, repo, common_path=None):
+    def iter_items(cls, repo, common_path=None, with_destination=False):
         """Equivalent to SymbolicReference.iter_items, but will return non-detached
         references as well."""
```

Comment:
Could you please describe what `with_destination` does in the doc-string? Since it is not available in the `SymbolicReference`, it's probably correct to put it here and make a note about this specialty in `SymbolicReference.iter_items()`.